docs/streams/developer-guide/dsl-api.html

```diff
@@ -3097,152 +3097,615 @@ <h5><a class="toc-backref" href="#id34">KTable-KTable Foreign-Key
+            <li>
+                Rewrite the return type of the renamed method to <code>void</code> and forward the record through the context;
+                and finally
+            </li>
+            <li>
+                Change the <code>KStream</code> call of the <code>transform</code> method to <code>process</code> or
+                <code>processValues</code>.
+            </li>
+        </ol>
+        <h3>Migration Examples</h3>
+        <p>
+            To migrate from the deprecated <code>transform</code>, <code>transformValues</code>, <code>flatTransform</code>, and
+            <code>flatTransformValues</code> methods to the Process API (PAPI) in Kafka Streams, follow these examples. The new
+            <code>process</code> and <code>processValues</code> APIs enable a more flexible and reusable approach by requiring
+            implementations of the <code>Processor</code> or <code>FixedKeyProcessor</code> interfaces.
+        </p>
+        <p>Here are examples to help you migrate:</p>
+        <table>
+            <thead>
+            <tr>
+                <th>Example</th>
+                <th>Migrating from</th>
+                <th>Migrating to</th>
+                <th>State Type</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr>
+                <td><a href="#cumulative-discounts-for-a-loyalty-program">Cumulative Discounts for a Loyalty Program</a></td>
+                <td><code>transform</code></td>
+                <td><code>process</code></td>
+                <td>Stateful</td>
+            </tr>
+            <tr>
+                <td><a href="#categorizing-logs-by-severity">Categorizing Logs by Severity</a></td>
+                <td><code>flatTransform</code></td>
+                <td><code>process</code></td>
+                <td>Stateless</td>
+            </tr>
+            <tr>
+                <td><a href="#traffic-radar-monitoring-car-count">Traffic Radar Monitoring Car Count</a></td>
+                <td><code>transformValues</code></td>
+                <td><code>processValues</code></td>
+                <td>Stateful</td>
+            </tr>
+            <tr>
+                <td><a href="#replacing-slang-in-text-messages">Replacing Slang in Text Messages</a></td>
+                <td><code>flatTransformValues</code></td>
+                <td><code>processValues</code></td>
+                <td>Stateless</td>
+            </tr>
+            </tbody>
+        </table>
+        <h4>Stateless Examples</h4>
+        <h5 id="categorizing-logs-by-severity">Categorizing Logs by Severity</h5>
+        <ul>
+            <li>
+                <strong>Idea:</strong> You have a stream of log messages. Each message contains a severity level (e.g., INFO,
+                WARN, ERROR) in the value. The processor filters messages, routing ERROR messages to a dedicated topic and
+                discarding INFO messages. The rest (WARN) are forwarded to another processor.
```

Comment:
> The rest (WARN) are forwarded to another processor.

I don't see this in the example? Seem `WARN` is going into some other topic, using a `TopicNameExtractor` ?